src/Network/Network.Test/SessionRecords/Commands.Network.Test.ScenarioTests.AzureFirewallTests/TestAzureFirewallCRUD.json

```diff
@@ -1252,10 +1252,10 @@
         ],
         "Date": [
           "Sat, 09 Mar 2019 22:38:08 GMT"
-        ],
+      
+        ],  ],
```

Comment:
Looks like a serialization error with an incorrect character here

Reply:
This is most likely what is causing your test failure.

Reply:
Fixed